mathics/builtin/__init__.py

```diff
@@ -148,15 +148,18 @@ def name_is_builtin_symbol(module, name: str) -> Optional[type]:
     # via the inspect module which is not implementation specific.
 
     # Skip those builtins defined in or imported from another module.
-    if module_object.__module__ != module.__name__:
-        return None
+    # if module_object.__module__ != module.__name__:
+    #     return None
```

Comment:
It seems that the check was added for a consistency check, and it is now preventing pymathics routines from loading.

I don't think we should be checking names. What exactly does this prevent?